### PR TITLE
Added default pair

### LINKED ISSUE #231: Default pair in the pair selector

### PR DESCRIPTION
**Pull request by nguvictor**
fixes #231

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_NETWORK=stokenet # Options: mainnet or stokenet
-NEXT_PUBLIC_DAPP_DEFINITION_ADDRESS=account_tdx_2_129kev9w27tsl7qjg0dlyze70kxnlzycs8v2c85kzec40gg8mt73f7y
\ No newline at end of file
+NEXT_PUBLIC_DAPP_DEFINITION_ADDRESS=account_tdx_2_129kev9w27tsl7qjg0dlyze70kxnlzycs8v2c85kzec40gg8mt73f7y
+NEXT_PUBLIC_DEFAULT_PAIR_ADDRESS=
```
Comment (EvgeniiaVak) [inline .env.example:3]: @nguvictor could you please add the stokenet address here? to be consistent with the rest of the file

**Review by EvgeniiaVak — CHANGES_REQUESTED** [review]
We might also want to have .env.mainnet and .env.testnet committed in the repo instead of one .env.example to make it easier to discover the vars, (both for new contributors and if the old ones set up the env on new machines or something)

**Review by EvgeniiaVak — APPROVED** [review]

**Comment by EvgeniiaVak** [comment]
@Radstakes could you please review this PR as well?

For it to work on deployed envs, you should also add the new variables:


from [@nguvictor's message](https://discord.com/channels/1125689933735145503/1125689934251032584/1200237007205576795) 
> Here are the addresses for each env:
> mainnet: component_rdx1czgjmu4ed5hp0vn97ngsf6mevq9tl0v8rrh2yq0f4dnpndggk7j9pu
> stokenet: component_tdx_2_1crs8ud8rr680krgtlskauye7qnns5zdawdlspvcqceder6tysu884p

**Comment by Radstakes** [comment]
Hi guys, added variables for default_pair to our Netlify config:

image

**Review by Radstakes — APPROVED** [review]
Default pair variables added to Netlify. All looks OK

**Pull request merged**